modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/construction/ConstructionService.scala

```diff
@@ -120,8 +121,9 @@ object ConstructionService {
           }.flatMap { proof =>
             EitherT
               .fromEither(
-                KryoSerializer[F]
+                JsonBinarySerializer
                   .serialize(Signed[DAGTransaction](transaction, NonEmptySet.of(proof)))
+                  .asRight[Throwable]
               )
               .leftMap(_ => MalformedTransaction.asInstanceOf[ConstructionError])
```

Comment:
While you're here...
```suggestion
              .leftMap[ConstructionError](_ => MalformedTransaction)
```

Reply:
Changed.